x-pack/plugins/cases/server/authorization/index.ts

```diff
@@ -241,31 +258,31 @@ export const Operations: Record<ReadOperations | WriteOperations, OperationDetai
     docType: 'comments',
     savedObjectType: CASE_COMMENT_SAVED_OBJECT,
   },
-  [ReadOperations.GetComment]: {
-    ecsType: EVENT_TYPES.access,
-    name: ACCESS_COMMENT_OPERATION,
-    action: 'case_comment_get',
-    verbs: accessVerbs,
-    docType: 'comments',
-    savedObjectType: CASE_COMMENT_SAVED_OBJECT,
-  },
-  [ReadOperations.GetAllComments]: {
+};
+
+/**
+ * Definition of all APIs within the cases backend.
+ */
+export const Operations: Record<ReadOperations | WriteOperations, OperationDetails> = {
```

Comment:
This looks worse than it is. I just moved out the `case`, `configuration`, and `attachment` operations in separate objects to reduce the size of the combined one.